index.html

```diff
@@ -1238,6 +1238,369 @@ <h2>Evidence</h2>
+	"issuer": "https://dmv.example.gov/issuers/14",
+	"issued": "2010-01-01",
+	"claim": {
+		"id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+		"name": "Mr John Doe",
+		"address": "10 Some Street, Anytown, ThisLocal, Country X",
+		"dateOfBirth": "2001-01-01",
+		"gender": "male",
+		"placeOfBirth": "Anytown, ThisLocal, Country X",
+		"nationality": "X",
+	},
+	"proof": { ...
+	}
+}
+        </pre>
+
+
+   </section>
+
+    <section>
+       <h2>Subject Delegates to Holder</h2>
+
+     <p>
+This is supported in the data model in two different ways: through the use of either recursion or verifiable presentations. Both methods involve the subject issuing a new verifiable credential to the delegated holder.
+    </p>
+
+    <p>
+This process of recursion could continue without restriction. Therefore 
+a new property “delegatable”, is added to the data model.
+
+     </p>
+
+   <section>
+
+   <h3> Delegatable Property </h3>
+
+  <p>
+The delegatable property, a Boolean, states whether a credential is 
+delegatable or not. If it is present with the value true, or not present, 
+then the credential is delegatable to a third party. If it is present with 
+the value false then it is not delegatable. A verifier that receives a verifiable 
+credential that is not delegatable MUST reject any verifiable credential that has
+been delegated from it.
+
+  </p>
+
+        <pre class="example nohighlight" title="An example of a credential that is not 
+delegatable">
+
+{
+	"id": "http://dmv.example.gov/credentials/3732",
+	"type": ["Credential", "ProofOfAgeCredential"],
+	"issuer": "https://dmv.example.gov/issuers/14",
+	"issued": "2010-01-01",
+	"delegatable": false,
+	"claim": {
+		"id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+		"ageOver": 21,
+	},
+	"proof": { ...
+	}
+}
+        </pre>
+
+       </section>
+       
+       <section>
+
+     <h3> Delegation through Recursion </h3>
+
+    <p>
+With recursion, the subject issues a credential to the delegated holder in which: 
+ the issuer is the subject, the claim contains 
+the id of the delegated holder, and the claim property is the credential that 
+was originally issued to the subject. An example is provided below.
+    </p>
+<p>
+Note: the exact format of this credential has not yet been agreed by the working group
+</p>
+ <pre class="example nohighlight" title="An example of a credential delegated 
+from a subject to a holder that is valid for 2 days">
+
+{
+	"id": "1234",
+	"type": ["Credential", "EmbeddedDelegatedCredential"],
+	"issuer": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+	"issued": "2010-01-06T00:00:00Z ",
+	"expires": "2010-01-08T00:00:00Z ",
+	"claim": {
+		"id": "did:example:ebfeb1f712ebc6f1c2123456789",
+		"credential": {
+			"id": "http://example.gov/credentials/3732",
+			"type": ["Credential", "Role"],
+			"issuer": "https://dmv.example.gov",
+			"issued": "2010-01-01",
+			"claim": {
+				"id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+				"role": "Project Manager"
```

Comment:
Why `role` and not `type`? ... and wouldn't you want to also associate their role w/ an organization and department? Again, this is why we shouldn't be expressing this sort of access control stuff as a VC... it's too easy to mess up.

Reply:
Because the property that is being delegated is the role.
Yes you could also associate organisation if needed. But typically the organisation will be the initial issuer who will issue roles to all its employees. And the organisation will be the root of trust for the verifier. So once an employee gets a role VC (issued by the organisation) and is allowed to delegate it to another employee, we do not really need the organisation name to be repeated in the delegated VC.

Reply:
I understand and now am more strongly against the proposal. This will lead to ambient authority attacks and we must not suggest that as a standard operating practice in the core specification: https://en.wikipedia.org/wiki/Ambient_authority

Reply:
I suggest that we do not support this use case in this version of the VC spec, noting that we may address it via OCAPs or in the next version of the specification and noting that delegation is very easy to get wrong and that extensions should be wary of any sort of delegation mechanism.

Reply:
Two points
1. By not mentioning delegation in the spec does not stop it from happening, but rather, allows unrestricted delegation to happen. This is what we learnt from v1 X.509, where anyone with an X.509 PKC could issue another PKC and thereby become a de-facto CA. We had to insert the basic constraints extension into v3 X.509 to stop this from happening. The Delegatable property that I am suggesting is equivalent to the X.509 basic constraints extension.
2. Role delegation does not introduce vulnerabilities of either the confused deputy or ambient authority types because the delegation of all privileges assigned to the role is intended. To quote Wikipedia  confused deputy is when "the associated permission changes unintentionally".